src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+type Props = {
+    journeys: JourneyParams[];
+};
+
+export function getJourneyUrl(journey: JourneyParams) {
+    return `/?from=${journey["fromStationId"]}&to=${journey["toStationId"]}&day=${journey["day"]}&time=${journey["time"]}`;
+}
+
+function getClass(time: JourneyParams) {
+    const times = time["time"];
```

Comment:
Minor point: using dot syntax (`time.time`) is typically preferred over square brackets (`time["time"]`) when the key is a valid JavaScript identifier (`time["something-with-special-characters"]` is an example of a key that is not).

Maybe the most idiomatic way to write this in JavaScript would be using destructuring syntax:
```ts
function getClass(params: JourneyParams) {
    const {time} = journeyParams;
    if (time === undefined) {
        ...
    }
    ...
}
```